ktane.js

```diff
@@ -1317,22 +1317,22 @@ angular.module('ktane', [])
                 7: { filename: "23-leftc.png", names: ["Backwards C with a dot", "Backwards C"], columns:[1, 2]},
                 8: { filename: "16-euro.png", names: ["Backwards E", "Backwards E with two dots"], columns:[2, 6]},
                 9: { filename: "26-cursive.png", names: ["Curly C"], columns:[2, 3]},
-                10: {filename: "3-hollowstar.png", names: ["Star outline", "Star"], columns:[2, 3]},
-                11: {filename: "20-questionmark.png", names: ["Upside down question mark", "Upsidedown questionmark"], columns:[2, 4]},
-                12: {filename: "1-copyright.png", names: ["Copyright mark", "copyright symbol"], columns:[3]},
-                13: {filename: "8-pumpkin.png", names: ["Saggy tits"], columns:[3]},
+                10: {filename: "3-hollowstar.png", names: ["Star outline", "Star", "Empty star"], columns:[2, 3]},
```

Comment:
It loops through them in order for a match, so this will already be matched by the 'star' beforehand.

Reply:
In which case, would it make sense to just start with "star" and have no other words?

Reply:
It depends whether we want it to read back 'star', or 'star outline' and 'star filled'.

Reply:
I think "star" saves a few milliseconds, and makes it simpler to read if the matching is all the same